src/main/webapp/script.js

```diff
@@ -114,7 +114,11 @@ let selectedIndex;
 
 function updateRestaurantList(restaurants) {
   let resContainerElement = document.getElementById("result-container");
-  resContainerElement.style.display = "block";
+
+  resContainerElement.classList.remove("hide");
```

Comment:
nice work fixing this. however note that im going to remove this after shifting the icons to the restaurant section.